src/AutoMapper/IMapperConfigurationExpression.cs

```diff
@@ -11,7 +11,8 @@ public interface IMapperConfigurationExpression : IProfileExpression
         /// <summary>
         /// Create missing type maps during mapping, if necessary
         /// </summary>
-        bool? CreateMissingTypeMaps { get; set; }
+        [Obsolete("Support for automatically created maps will be removed.")]
```

Comment:
Let's include the version where it will be removed, 9.0, and include the fix, "you will need to explicitly configure maps".